src/HttpRequest.cc

```diff
@@ -738,3 +738,40 @@ HttpRequest::manager(const CbcPointer<ConnStateData> &aMgr, const AccessLogEntry
     }
 }
 
+/// \returns a pointer to known *_port address where the request was accepted
```

Comment:
same \returns problem as mentioned in .cc file.

To avoid re-writing the same text as the other function use this as a template:
  /// ... text about unique things in this function (eg parameters) ...
  /// \see const Ip::Address *FindListeningPortAddress(const HttpRequest *, const AccessLogEntry *)

Reply:
Done